lib/semantics/check-stop.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2019, Arm Ltd.  All rights reserved.
```

Comment:
Paul, we need your permission to relicense this under the new llvm license.  We'll do the relicensing soon.

Reply:
How this (giving permission) will work in practice? We have in-house approval for contributions to specific project given specific license. Did you discuss this with Richard during EuroLLVM? What about Kiran's commits?

Reply:
Kiran's commits didn't have copyrights, did they?

How does this work?  You just say it's ok.

Reply:
so I say it's OK to relicense files added by me to a compatible open-source license.